customize.dist/translations/messages.es.js

```diff
@@ -18,7 +18,7 @@ define(function () {
     out.synchronizing = "Sincronización";
     out.reconnecting = "Reconectando...";
     out.lag = "Retraso";
-    out.readonly = 'Solo lectura';
+    out.readonly = 'Sólo lectura';
```

Comment:
Thank you for reviewing.

You're right, the Spanish language authority (RAE) deprecated the use of "sólo" a few years ago. It's funny because I trusted my Windows 7 installation, which reads "Sólo lectura", but the truth is that Windows 10 has already updated this piece of text with the new rule, so that it now reads "Solo lectura".

I don't know whether RAE rules are applicable to other Spanish variants different from es_ES.

To sum up: I think that the accent should be removed.